### ISSUE TITLE
Genie.newapp("MyGenieApp") permission denied

### ISSUE THREAD

**Issue by huaxk** [GenieFramework/Genie.jl#147]
**Describe the bug**
When I create a new app using Genie, I get error:
```
julia> using Genie
julia> Genie.newapp("MyGenieApp")
ERROR: SystemError: opening file "MyGenieApp\\bootstrap.jl": Permission denied    
Stacktrace:                                                                        
 [1] #systemerror#43(::Nothing, ::Function, ::String, ::Bool) at .\error.jl:134    
 [2] systemerror at .\error.jl:134 [inlined]                                       
 [3] #open#309(::Nothing, ::Nothing, ::Nothing, ::Bool, ::Nothing, ::Function, ::St
ring) at .\iostream.jl:283                                                         
 [4] #open at .\none:0 [inlined]                                                   
 [5] open(::String, ::String) at .\iostream.jl:339                                 
 [6] #open#310(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}
}}, ::Function, ::getfield(Genie.REPL, Symbol("##4#7")){Tuple{String,String}}, ::St
ring, ::Vararg{String,N} where N) at .\iostream.jl:367                             
 [7] open at .\iostream.jl:367 [inlined]                                           
 [8] #newapp#1(::Bool, ::Bool, ::Bool, ::Function, ::String) at C:\Users\Administra
tor\.julia\packages\Genie\QqqnQ\src\REPL.jl:57                                     
 [9] #newapp at .\none:0 [inlined]                                                 
 [10] #newapp#6 at C:\Users\Administrator\.julia\packages\Genie\QqqnQ\src\Genie.jl:
74 [inlined]                                                                       
 [11] newapp(::String) at C:\Users\Administrator\.julia\packages\Genie\QqqnQ\src\Ge
nie.jl:74                                                                          
 [12] top-level scope at none:0      
```                                              

**To Reproduce**
I am on windows10, julia 1.1.0, Genie v0.9.4

**Additional context**
```
julia>  versioninfo()
Julia Version 1.1.0
Commit 80516ca202 (2019-01-21 21:24 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i5-3337U CPU @ 1.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, ivybridge)
Environment:
  JULIA = C:\Users\Administrator\AppData\Local\JuliaPro\Julia-1.0.3\bin\julia.exe
  JULIA_EDITOR = code
  JULIA_EXE = julia.exe
  JULIA_HOME = C:\Users\Administrator\AppData\Local\JuliaPro\Julia-1.0.3\bin
  JULIA_PKG_DEVDIR = C:/sources/dev
```
```
(v1.1) pkg> st
    Status `C:\Users\Administrator\.julia\environments\v1.1\Project.toml`
   [c43c736e] Genie v0.9.4
```

**Comment by essenciary** [comment]
@huaxk Did you install the registered version? Ie `pkg> add Genie`? I noticed that the registered version has different permissions. I'm working on an update, till then please go with the #master (`pkg> add https://github.com/ GenieFramework/Genie.jl`)

**Comment by huaxk** [comment]
@essenciary Yes,I am install the registered version，the above problem appeared.  Then I installed the #master and it was ok！Thanks!

**Comment by essenciary** [comment]
@huaxk No worries - I'll register a new version which sets the proper permissions and I'll open an issue as the change in permissions doesn't seem right.

**Comment by essenciary** [comment]
@huaxk This is now fixed on registered v0.9.5
